digiwf-cypress/cypress/support/commands.js

```diff
@@ -33,5 +33,7 @@ Cypress.Commands.add("loginUser", () => {
     cy.get("#username").type(username);
     cy.get("#password").type(password);
     cy.get('[id^=kc-login]').click();
-})
-
+    if(home.toString().includes("local")){
+        cy.contains("http://keycloak:8080/auth/realms/P82").click();
+    }
```

Comment:
Den verstehe ich nicht😅

Reply:
dieser Klick funktioniert nicht in unserer internen Testumgebung (ist auch nicht notwendig), wenn ich das ohne das if ausführe schlägt die interne Pipe fehl, wenn ich es gar nicht ausführe schlägt das lokale fehl

Reply:
```suggestion
    if(home.toString().includes("local")){
        // Only necessary for local, since slightly different keycloak setup
        cy.contains("http://keycloak:8080/auth/realms/P82").click();
    }
```